docs/arguments.md

```diff
@@ -37,6 +37,14 @@
 - `-distributor.extra-query-delay`
    This is used by a component with an embedded distributor (Querier and Ruler) to control how long to wait until sending more than the minimum amount of queries needed for a successful response.
 
+## Ingester
+
+- `-ingester.normalise-tokens`
+
+   Write out "normalised" tokens to the ring.  Normalised tokens consume less memory to encode and decode; as the ring is unmarshalled regularly, this significantly reduces memory usage of anything that watches the ring.
+
+   Before enabling, rollout a version of Cortex that supports normalised token (ie ones which have this flag) for all jobs that interact with the ring, then rollout this flag set to `true`.  The new ring code can still read and write the old ring format, so is backwards compatible.
```

Comment:
Does this mean queriers and distributors have a `-ingester...` flag?

Reply:
No, the ingesters are the only thing that write to the ring, and the only thing that needs this flag.  Have clarified/